src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -62,6 +81,43 @@ void check_sshfs_exists(mp::SSHSession& session)
     run_cmd(session, "which sshfs", error_handler);
 }
 
+// If the target name starts with ~, this function returns the full directory name. If not, it returns the same
+// target name.
+std::string expand_home_directory(mp::SSHSession& session, const std::string& target)
+{
+    std::string expanded_target;
+
+    if ('~' == target[0])
+    {
+        std::string home;
+
+        // Get the user name, which can be empty.
+        auto pos = (1 == target.size()) ? 1 : target.find('/', 1);
+        if (1 == pos)
+        {
+            home = run_string_cmd(session, "pwd");
+        }
+        else
+        {
+            std::string username = target.substr(1, pos - 1);
+            home = run_string_cmd(session, "getent passwd " + username + " | cut -d : -f 6");
```

Comment:
Any reason to not use `fmt::format`?

Reply:
No reason :)
From the pdf you sent, spaces are not allowed.

Reply:
> spaces are not allowed.

…by default :)